backend/drm/drm.c

```diff
@@ -1032,7 +1050,7 @@ static void realloc_crtcs(struct wlr_drm_backend *drm) {
 		connector_match[i] = -1;
 	}
 	for (size_t i = 0; i < drm->num_crtcs; ++i) {
-		if (new_match[i] != UNMATCHED) {
+		if (new_match[i] != UNMATCHED && new_match[i] != SKIP) {
```

Comment:
Without this sway would crash when the lease is terminated, but I'm not 100% sure this is the right fix rather than a band-aid. Probably a consequence of the logic in the previous hunk, which may also be incorrect. Seems to work but w/e. Paging @ascent12 for comment.

Reply:
Actually, this is definitely wrong.

Reply:
Found the issue, it was elsewhere